src/main/java/frc/robot/robotcontainers/MiniSwerveContainer.java

```diff
@@ -48,7 +49,13 @@ public MiniSwerveContainer() {
   private void configureButtonBindings() {
     Joystick driver = new Joystick(0);
 
-    SwerveSubsystem.getInstance().setDefaultCommand(new SwerveTeleop(driver));
+    SwerveSubsystem.getInstance().setDefaultCommand(new SwerveTeleop(driver, Config.Swerve.teleopSpeed, Config.Swerve.teleopAngularSpeed,5.0));
+    
+    new JoystickButton(driver, XboxController.Button.kLeftBumper.value).whenHeld(
+      new SwerveTeleop(driver, Config.Swerve.teleopFastSpeed, Config.Swerve.teleopFastAngularSpeed,5.0));
+
+    new JoystickButton(driver, XboxController.Button.kRightBumper.value).whenHeld(
+      new SwerveTeleop(driver, Config.Swerve.teleopSlowSpeed, Config.Swerve.teleopSlowAngularSpeed,20.0));
```

Comment:
Rate limits should be moved to Config eventually. That can be done in a different pull request since this pull request needs to be accepted before Saturday to avoid slowing down anything then.